src/DotNetStatus/Controllers/DataController.cs

```diff
@@ -0,0 +1,202 @@
+﻿using DotNetStatus.Common;
+using Microsoft.AspNet.Hosting;
+using Microsoft.AspNet.Mvc;
+using Microsoft.AspNet.Mvc.Rendering;
+using Microsoft.Framework.Configuration;
+using Microsoft.Fx.Portability;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DotNetStatus.Controllers
+{
+    [Route("Data")]
+    public class DataController : Controller
+    {
+        private readonly string _apiPortServiceUrl;
+
+        public DataController(IConfiguration config, IHostingEnvironment env)
+        {
+            if (env.IsDevelopment())
+            {
+                _apiPortServiceUrl = config["ApiPortService:DebugSiteUrl"];
```

Comment:
You should not need to pass the config around to get the apiportservice url. This should be encapsulated in the ApiPortClient